tests/run_code_style.sh

```diff
@@ -4,13 +4,11 @@ set -xeu
 
 if [ $1 = "lint" ]; then
     flake8 ignite tests examples --config setup.cfg
-    isort . --check --settings setup.cfg
-    black . --check --config pyproject.toml
+    ufmt diff .
```

Comment:
@jreese i wonder why here you use `ufmt diff .` and in `tests/run_code_style.bat` : `ufmt check .` ?

Reply:
Ah, I originally had it as check in both places to match existing behavior, but switched it to diff when I was working out the correct configuration. I personally think diff is more useful, and would suggest using that in both places, but would defer that decision to your opinion.

Reply:
I see. If they both behave similarly in CI and diff shows the diff. I'm ok with `ufmt diff .` everywhere.